core/src/main/scala/io/finch/ToService.scala

```diff
@@ -98,17 +97,19 @@ object ToService {
       }
   }
 
+  type IsCoproduct[C] = OrElse[C <:< Coproduct, DummyImplicit]
```

Comment:
Is this type-alias for readability or it doesn't work w/o it?

Reply:
Just for the sake of readability, yes.

Reply:
👍 Maybe rename it to `IsNegotiable` so it make sense in our domain.